app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -55,8 +64,15 @@ public GameView(Context context, int screenX, int screenY, boolean isPlaying) {
         // Margin in px
         screenMargin = (int) getResources().getDimension(R.dimen.fab_margin);
 
+        // Handpicked value
```

Comment:
Isn't it explained in the next line?

Reply:
why 5 * 3?

Reply:
I was trying to find the ratio of the grass height to the platform image height. It is not accurate :) Not sure that this is the best explanation and solution. I can define a constant "GRASS_HEIGHT" for example (which I can calculate).

Reply:
oh I see. We can have this for now. But it maybe better to move the computation at the top of the class into a variable. You can reuse the same comment but it would be easier to change in the future